packages/kbn-monaco/src/console/lexer_rules/shared.ts

```diff
@@ -14,11 +14,13 @@ export const consoleSharedLanguageConfiguration: monaco.languages.LanguageConfig
   brackets: [
     ['{', '}'],
     ['[', ']'],
+    ['"""', '"""\n'],
```

Comment:
This makes the editor apply correct indentation when the user presses Enter between the opening and the closing `"""`.